src/main/resources/swagger/api-docs.yaml

```diff
@@ -1643,6 +1643,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+    put:
```

Comment:
This feels more like a PATCH to me than a POST. We're updating the value, not creating it. Well, it may be being created if going from 0 -> 1 groups, but more than that is an update instead of a creation.